src/config.h

```diff
@@ -15,7 +15,10 @@
 	#define PROJECT_NAME "minetest"
 	#define PROJECT_NAME_C "Minetest"
 	#define STATIC_SHAREDIR ""
-	#include "android_version.h"
+	#define VERSION_MAJOR 5
+	#define VERSION_MINOR 1
+	#define VERSION_PATCH 0
```

Comment:
Information duplication. The version bump script needs to be adjusted to replace these lines.